docs/framework/data/adonet/ef/language-reference/query-plan-caching-entity-sql.md

```diff
@@ -22,7 +22,7 @@ Whenever an attempt to execute a query is made, the query pipeline looks up its
   
  Query plan caching on static queries and parameterized queries can provide performance benefits. The following is an example of a static query:  
   
-```  
+```sql  
```

Comment:
```suggestion
```csharp
```

seems like this one should be c#? it would be good to convert the code on line 19 to a code block as well.

Reply:
This isn't a complete C# line. It would be a valid SQL if the double quotes and the semi-colon are removed. Does that seem good, @mairaw ?

Reply:
@mairaw I implemented the suggestion from @Youssef1313 and made these csharp.